qp_deblur/deblur.py

```diff
@@ -321,6 +321,16 @@ def generate_insertion_trees(placements, out_dir,
                      % (file_ref_rename, std_out, std_err))
         raise ValueError(error_msg)
 
+    # this is required cause there is a missing branch in the SEPP tree
```

Comment:
can we be even more precise: the branch length is missing in Siavash's Greengenes 13.8 99% reference tree (which is our default) and this missing branch length is propagated to all generated insertion trees.